custom/debugcheats.user.js

```diff
@@ -0,0 +1,319 @@
+// ==UserScript==
+// @name          [Pokeclicker] Debug Cheats
```

Comment:
The name needs to be following the 3 word naming schema, just like all the other scripts.